allen-cell-animated/nucmorph-colorizer
# doc: Update README documentation links, remove `documentation` folder

Problem
=======
*Estimated review size: small, 5-10 minutes*

Deletes the `documentation` folder since it is duplicated from [`colorizer-data`](https://github.com/allen-cell-animated/colorizer-data) and required us to sync two different sets of documentation.

Solution
========
- Updates documentation links to point to `colorizer-data` repo.
- Deletes the `documentation` folder.
- Renames references to "Nucmorph Colorizer" to "Timelapse Feature Explorer"
- Slightly updates the core shader for readability.

## Type of change
* This change requires a documentation update

Steps to Verify:
----------------
1. See new README here: https://github.com/allen-cell-animated/nucmorph-colorizer/tree/doc/update-links
